blog/blog.sql

```diff
@@ -0,0 +1,5 @@
+create table blogo (
+    id serial primary key,
+    username text,
+    posts timestamp
```

Comment:
You might want to have two tables, one for `users` and another for `posts` (with `posts` having a foreign key to the `users` table).

Reply:
```create table users (
    userId serial primary key,
    username text
);

create table posts
(
    userId serial primary key,
    post text,
    post_time timestamp
);

would this be more appropriate, and i'm assuming i'd want to name my id something that would indicate what the id is used for?